package.json

```diff
@@ -69,7 +69,7 @@
     "react-ga": "^2.1.2",
     "react-select": "^1.0.0-rc.5",
     "react-virtualized-select": "^3.1.0",
-    "react2angular": "^1.1.3",
+    "react2angular": "^2.0.0",
```

Comment:
anything different in 2.0 or just keeping up with dependencies?

Reply:
They did have a change regarding scope changes that I was hoping would stop me from having to $applyAsync, but I still had to do it. Left the update in anyways.